Add the instruction for enable API audit log in downstream clusters (#824)
rancher/rancher-docs: docs/how-to-guides/advanced-user-guides/enable-api-audit-log-in-downstream-clusters.md

Author: Note to reviewers:

I could not figure out how to show this page under the `How-to Guides -> Advanced User Guides` section. 
Could you provide some guidance?

Reviewer: Sure. It sounds like an issue with the sidebar. We've been working on a PR to add that kind of information to the repo:

> Docusaurus generates sidebars based on a JSON file named sidebars.js. When you add a new doc, you need to add an entry to sidebars.js. Otherwise, the page won't appear in the list of sidebar topics. 

> The sidebars.js file for the latest version of Rancher is located in the top level of the repo. Versioned docs each have their own versioned sidebar, found within /versioned_sidebars in the top level of the repo.

> The schema for `sidebars.js` looks like this: 

```JS
sidebar: [
    "toplevel",
    {
        type: "category",
        label: "Second Level",
        items: [
            "second-level/overview",
            {
                type: "category",
                label: "Topic One",
                link: {
                    type: "doc",
                    id: "pages-for-subheaders/index-page-for-topic-one"
                }
                items: [
                    "second-level/topic-one/page-a",
                    "second-level/topic-one/page-b",
                ]
            }
        ]
    }
]
```

> Paths for docs files are listed within an items array. Their position within the file is similar to how they'll be organized within the published sidebar.

Author: It works now. thank you

---

> title: Enable API Audit Log in downstream clusters 

Author: fixed.

---
title: Enabling the API Audit Log in Downstream Clusters 
---

<head>
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-api-audit-log-in-downstream-clusters"/>
</head> 

> Kubernetes auditing provides a security-relevant chronological set of records about a cluster. Kube-apiserver performs auditing. Each request on each stage of its execution generates an event, which is then pre-processed according to a certain policy and written to a backend. The policy determines what’s recorded and the backends persist the records.

Reviewer 2: ```suggestion
Kubernetes auditing provides a security-relevant chronological set of records about a cluster. Kube-apiserver performs auditing. Requests generate an event at each stage of its execution, which is then preprocessed according to a certain policy and written to a backend. The policy determines what’s recorded and the backend persists the records.
```

Author: fixed

> You might want to configure the audit log as part of compliance with the CIS (Center for Internet Security) Kubernetes Benchmark controls.

Reviewer 2: ```suggestion
You might want to configure the audit log as part of compliance with the Center for Internet Security (CIS) Kubernetes Benchmark controls.
```

Author: Fixed.
